lightly_studio/src/lightly_studio/examples/example_operators.py

```diff
@@ -0,0 +1,76 @@
+"""Example of how to add samples in coco caption format to a dataset."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+from uuid import UUID
+
+from environs import Env
+from sqlmodel import Session
+
+import lightly_studio as ls
+from lightly_studio import db_manager
+from lightly_studio.plugins.base_operator import BaseOperator, OperatorResult
+from lightly_studio.plugins.operator_registry import operator_registry
+from lightly_studio.plugins.parameter import BaseParameter, BoolParameter, StringParameter
+
+
+@dataclass
+class TestOperator(BaseOperator):
+    """Dummy Operator for demo purpose."""
+
+    name: str = "test operator"
+    description: str = "used to test the operator and registry system"
+
+    @property
+    def parameters(self) -> list[BaseParameter]:
+        """Return the list of parameters this operator expects."""
+        return [
+            BoolParameter(name="test flag", required=True),
+            StringParameter(name="test str", required=True),
+        ]
+
+    def execute(
+        self,
+        *,
+        session: Session,
+        dataset_id: UUID,
```

Comment:
Does dataset_id still make sense in the multimodal setup? Is this the "root dataset id"?